cms/djangoapps/contentstore/tests/test_crud.py

```diff
@@ -36,7 +36,7 @@ def test_get_templates(self):
         self.assertIn('markdown', dropdown['metadata'])
         self.assertIn('data', dropdown)
         self.assertRegexpMatches(dropdown['metadata']['markdown'], r'^Dropdown.*')
-        self.assertRegexpMatches(dropdown['data'], r'<problem>\s*<p>Dropdown.*')
+        self.assertRegexpMatches(dropdown['data'], r'<problem>\s*<question>\s*<p>Dropdown.*')
```

Comment:
Perhaps not part of this PR, but it might be to our benefit for maintenance to not use regexes for comparing XML.

Django has `assertXMLEqual` (https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.SimpleTestCase.assertXMLEqual) and also `assertInHTML`, neither of which quite fit this usecase, sadly.